src/main/java/com/ratifire/devrate/controller/FeedbackController.java

```diff
@@ -0,0 +1,34 @@
+package com.ratifire.devrate.controller;
+
+import com.ratifire.devrate.dto.FeedbackDto;
+import com.ratifire.devrate.exception.FeedbackSubmissionLimitException;
+import com.ratifire.devrate.service.FeedbackService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller class responsible for handling requests related to feedback information.
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/feedbacks")
```

Comment:
it shoud be {userId}/feedbacks

Reply:
In that case it should be in the UserController.
Are you suggesting calling the FeedbackService from the UserController?